articles/app-service/overview-managed-identity.md

```diff
@@ -443,9 +447,22 @@ For .NET applications and functions, the simplest way to work with a managed ide
     // OR
     var kv = new KeyVaultClient(new KeyVaultClient.AuthenticationCallback(azureServiceTokenProvider.KeyVaultTokenCallback));
     ```
+If you want to use a user-defined managed identity, then you'll need to use the following code instead modifying to target the right resources and use the right clientid:
+
+    ```csharp
+    using Microsoft.Azure.Services.AppAuthentication;
+    using Microsoft.Azure.KeyVault;
+    // ...
+    var azureServiceTokenProvider = new AzureServiceTokenProvider("RunAs=App;AppId=<clientid-guid>");
+    string accessToken = await azureServiceTokenProvider.GetAccessTokenAsync("https://vault.azure.net");
+    // OR
+    var kv = new KeyVaultClient(new KeyVaultClient.AuthenticationCallback(azureServiceTokenProvider.KeyVaultTokenCallback));
+    ```
 
 To learn more about Microsoft.Azure.Services.AppAuthentication and the operations it exposes, see the [Microsoft.Azure.Services.AppAuthentication reference] and the [App Service and KeyVault with MSI .NET sample](https://github.com/Azure-Samples/app-service-msi-keyvault-dotnet).
 
+To learn more about how to configure the AzureServiceTokenProvider for service to service authentication, see the [Service to Service Authentication with .NET documentation](https://docs.microsoft.com/en-us/azure/key-vault/general/service-to-service-authentication#connection-string-support)
```

Comment:
This should be combined with the paragraph above.

Reply:
This thread can be resolved. I have created a suggestion comment capturing the change.